msteams-platform/promote-app-adoption.md

```diff
@@ -39,7 +39,7 @@ Admins can set up an app governance process that manages your organization's IT
 
 ## Understand how you can drive app adoption
 
-Following the below five easy steps, you can encourage customers to adopt your apps through admin recommendations to their users, for maximum adoption in an organization. Consider the following elements for app adoption.
+By following these five easy steps, you make it easy for your customers to adopt apps within their organizations.
```

Comment:
Can we rewrite the sentence as "You can help your customers to adopt apps within their organizations by following these steps".